index.html

```diff
@@ -5,44 +5,52 @@
             <meta http-equiv="X-UA-Compatible" content="IE=edge">
             <meta name="viewport" content="width=device-width, initial-scale=1.0">
             <link rel="stylesheet" href="style.css">
-            <title>simpleweb</title>
+            <title>Portfolioweb</title>
         </head> 
         <body>
             
         <header>
       <!--  navigation bar -->
             <nav class="toolbar">
                 <div class="logo"><a href="#">My Logo</a></div>
-                <div class="menu-icon"><img src="images/menu.png" alt="follow me on angellist" width="24" height="24" ></div>
+                <div class="nav-div">
+                    <ul class="nav-link">
+                        <li class="nav-link-item"><a href="#">Portfolio</a></li>
+                        <li class="nav-link-item"><a href="#">About</a></li>
+                        <li class="nav-link-item"><a href="#">Contact</a></li>
+                    </ul>
+                </div>
             </nav>
          </header>
             <!--  header section -->
              <section class="first-page">
```

Comment:
- [ ] [REQUIRED] Good job so far! Please fix your project `logo` stuck to the left and your `text` elements dropped to the bottom and wasted space at the `top`. Also, give the `border` on the left-bottom corner of your Web page a `border-radius` as shown in [Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A122&t=Uy7TAUKVJbGSY3Ol-0) 🙏🏼 

![image](https://user-images.githubusercontent.com/63915024/213461687-428f8028-d2b2-40b4-a282-dc570c1f1cae.png)